src/ServiceControlInstaller.Engine/Unattended/UnattendServiceControlSplitter.cs

```diff
@@ -0,0 +1,187 @@
+                case RequiredUpgradeAction.ConvertToAudit:
+                    return Result.Failed("This instance cannot have an Audit instance split from it. Convert the instance into an Audit instance instead", RequiredUpgradeAction.ConvertToAudit);
+                case RequiredUpgradeAction.SplitOutAudit:
+                    return Result.Success;
+                default:
+                    return Result.Failed("This instance cannot have an Audit instance split from it. This instance has no recommended upgrade action");
+            }
+        }
+
+        Result ValidateUpgradeVersion(ServiceControlInstance instance)
+        {
+            var upgradeInfo = UpgradeControl.GetUpgradeInfoForTargetVersion(serviceControlInstaller.ZipInfo.Version, instance.Version);
+            if (instance.Version < upgradeInfo.CurrentMinimumVersion)
+            {
+                return Result.Failed($"An interim upgrade to version {upgradeInfo.RecommendedUpgradeVersion} is required before upgrading to version {serviceControlInstaller.ZipInfo.Version}. Download available at https://github.com/Particular/ServiceControl/releases/tag/{upgradeInfo.RecommendedUpgradeVersion}");
+            }
+
+            return Result.Success;
+        }
+
+        Result ValidateServiceAccount(Options options, ServiceControlAuditNewInstance auditDetails)
+        {
+            var account = UserAccount.ParseAccountName(auditDetails.ServiceAccount);
+            if (!account.CheckPassword(auditDetails.ServiceAccountPwd))
+            {
+                return Result.Failed(
+                    string.IsNullOrWhiteSpace(options.ServiceAccountPassword)
+                        ? $"A password is required for the service account: {auditDetails.ServiceAccount}"
+                        : $"Incorrect password for {auditDetails.ServiceAccount}");
+            }
+
+            return Result.Success;
+        }
+
+        Result ValidateLicense()
+        {
+            var licenseCheck = auditInstaller.CheckLicenseIsValid();
+            if (!licenseCheck.Valid)
+            {
+                return Result.Failed(licenseCheck.Message);
+            }
+
+            return Result.Success;
+        }
+
+        ServiceControlAuditNewInstance CreateSplitOutAuditInstance(ServiceControlInstance source)
+        {
+            return new ServiceControlAuditNewInstance
+            {
+                AuditQueue = source.AuditQueue,
+                AuditLogQueue = source.AuditLogQueue,
+                ForwardAuditMessages = source.ForwardAuditMessages,
+                AuditRetentionPeriod = source.AuditRetentionPeriod,
+                TransportPackage = source.TransportPackage,
+                ConnectionString = source.ConnectionString,
+                HostName = source.HostName,
+                Name = $"{source.Name}.Audit",
+                ServiceAccount = source.Service.Account,
+                // NOTE: This should always be blank
```

Comment:
hmmm what should be blank? ☝️

Reply:
The password. We never read it from the instance so it should always be blank